### DISCUSSION TITLE
Just upgraded from 23H2 to 24H2 successfully.

### DISCUSSION THREAD

**Discussion by bdog549** [AveYo/MediaCreationTool.bat#385] [General]
Just a FYI for anyone that sees this. I've used this bat file tool to go from 22H2 to 23H2 and just now upgraded to 24H2 successfully. As soon as I mounted the 24H2 ISO, and ran the setup.exe. The script ran automatically as it has done on previous upgrades, and  a command prompt opened for a few seconds, and the install process resumed. It still works fine. 24H2 is running well as far as I can tell -- no bugs or issues atm.

I can't recall which file I ran three years ago, either the auto.cmd or the skip TPM check on dynamic update. I'm guessing the latter.  If AveYo knows or anyone else please chime in so I can pass on the info. I just can't remember what I ran originally lol. I think when I originally installed W11, I did the process of deleting the appraiserres.dll file in the Windows BT Sources folder.

**Comment by adelphiaUK** [comment]
I hope you have more luck with it than those that are being **forced** to take it as a _Windows Downdate_.
